Catappult/appcoins-wallet-android
# [APPC-4465] Android integration amazon pay TopUp Flow

**What does this PR do?**

  Implement Amazon Pay integration in TopUp with both flows, the first purchase without token and the second purchase with token to make faster payments.

**Database changed?**

  No

**How should this be manually tested?**

  Enter in TopUp Flow and try to make a purchase with amazonPay and after first purchase try again to see the faster payments

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4465](https://aptoide.atlassian.net/browse/APPC-4465)

**Code Review Checklist**



[APPC-4465]: https://aptoide.atlassian.net/browse/APPC-4465?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
